### ISSUE TITLE
Wrong definition for RichTextDate

### ISSUE THREAD

**Issue by AlesDo** [soxtoby/SlackNet#189]
The rich text block date element does not correspond to the Slack BlockKit definition instead of text field it should have format field.
Here is a valid JSON from the Slack Block Kit Builder.
```JSON
{
	"blocks": [
		{
			"type": "rich_text",
			"elements": [
				{
					"type": "rich_text_section",
					"elements": [
						{
							"type": "date",
							"timestamp": 1705569650,
							"format": "{date_long_pretty} {time}"
						}
					]
				}
			]
		}
	]
}
```

It is not possible to recreate this in code since there is no format property and sending a message without format property fails because it is required. You can try the following snippet.

```CSharp
var response = await slackApiClient
    .WithAccessToken(slackExchangeResponse.AccessToken)
    .Chat.PostMessage(
        new Message()
        {
            Channel = channel,
            Text = "Test",
            Blocks =
                [
                    new RichTextBlock()
                    {
                        Elements = 
                            [
                                new RichTextSection()
                                {
                                    Elements = 
                                        [
                                            new RichTextDate()
                                            {
                                                Text = "{date_long_pretty} {time}", // this should be Format instead
                                                Timestamp = DateTime.Now.ToTimestamp()
                                            }
                                        ]
                                }
                            ]
                    }
                ]
            
        }
    );
```

**Comment by soxtoby** [comment]
Thanks for raising this, @AlesDo. I'll update `RichTextDate` and push up a new version shortly.

**Comment by soxtoby** [comment]
Fix is now available in [v0.12.2](https://github.com/soxtoby/SlackNet/releases/tag/v0.12.2).
